src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/CachePoolClear/config.yml

```diff
@@ -0,0 +1,16 @@
+imports:
+    - { resource: ../config/default.yml }
+
+services:
+    dummy1:
+        class: Symfony\Bundle\FrameworkBundle\Tests\Fixtures\DeclaredClass
+        arguments: ['@cache.private_pool']
+    dummy2:
+        parent: dummy1
```

Comment:
@nicolas-grekas To clarify, removing this alias make the  `cache.private_pool` unclearable, neither from `cache:clear` nor from `cache:pool:clear`, and it make the `testClearPrivatePool` test fail. I tended to think it's a bug, but maybe I'm wrong